docs/source/_toctree.yml

```diff
@@ -280,7 +280,9 @@
       title: UniSpeech
     - local: model_doc/unispeech-sat
       title: UniSpeech-SAT
-    - local: model_doc/vision-encoder-decoder
+    - local: model_doc/vilt
+      title: ViLT
+    - local: model_doc/visionencoderdecoder
```

Comment:
This line should not be changed. Each model doc page should have the same name as the model type.